callback.go

```diff
@@ -242,11 +246,40 @@ func callbackArg(typ reflect.Type) (callbackArgConverter, error) {
 	case reflect.Float32:
 		c := callbackArgCast{callbackArgFloat64, typ}
 		return c.Run, nil
+	case timeKind:
+		return callbackArgTime, nil
 	default:
 		return nil, fmt.Errorf("don't know how to convert to %s", typ)
 	}
 }
 
+func callbackArgTime(v *C.sqlite3_value) (reflect.Value, error) {
+	if C.sqlite3_value_type(v) != C.SQLITE_TEXT {
+		return reflect.Value{}, fmt.Errorf("argument must be an TEXT")
+	}
+	var t time.Time
+	var err error
+	c := (*C.char)(unsafe.Pointer(C.sqlite3_value_text(v)))
+	tv := C.GoString(c)
+	s := strings.TrimSuffix(tv, "Z")
```

Comment:
The sqlite3_value is coming from the database. So can I assume it will not have timezone in it ?

Reply:
https://www.sqlite.org/lang_datefunc.html

SQLite does support time strings with time zone.

See section `Time Strings`